test/Mvc6Framework45.FunctionalTests/bower.json

```diff
@@ -2,11 +2,9 @@
   "name": "ASP.NET",
   "private": true,
   "dependencies": {
-    "bootstrap": "3.0.0",
-    "jquery": "1.10.2",
-    "jquery-validation": "1.11.1",
-    "jquery-validation-unobtrusive": "3.2.2",
-    "hammer.js": "2.0.4",
-    "bootstrap-touch-carousel": "0.8.0"
+    "bootstrap": "3.3.5",
```

Comment:
Per our discussion, you can probably remove all of the JS libs etc. since we don't need them for these unit tests.